lib/div-alg.gi

```diff
@@ -2803,4 +2818,4 @@ od;
 
 return fail;
 
-end;
+end;
```

Comment:
Another whitespace change - your version has no new line at the end of the file. Please restore it.